### PR TITLE
protect create nft page from user unregistered

### PR DESCRIPTION
**Pull request by Derick1530**
# PR Summary

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by eliezerbasubi** [comment]
@Derick1530 kindly resolve merge conflicts

File: components/modules/__noAuth/Header/index.tsx [hunk 1]
```diff
@@ -35,7 +34,9 @@ const Header = () => {
   };
 
   const onCreateNft = () => {
-    routes.push("/create");
+    if (address) {
+      routes.push("/create");
+    }
   };
```
[the remark this answers is not in the document]
Comment (Derick1530) [reply components/modules/__noAuth/Header/index.tsx]: Now it's working

[the remark this answers is not in the document]
Comment (Derick1530) [reply components/modules/__noAuth/Header/index.tsx]: > @Derick1530 Can you try to upgrade the packages and try again?

@eliezerbasubi  Now it's working

**Comment by eliezerbasubi** [comment]
@Derick1530 Kindly resolve conflicts so that we can merge this PR.

**Comment by Derick1530** [comment]
dan
@danmugh I saw you merge a PR where you add these lines, 
as you can see I'm getting errors when trying to push. Any help from you will be appreciated

**Comment by danmugh** [comment]
Can you please resolve conflicts bro

**Comment by Derick1530** [comment]
> Can you please resolve conflicts bro

Conflicts are already been solved, the thing is I got an error as explained above when I'm trying to push the changes which will fix the conflicts.

**Pull request closed without merging**